sites/platform/src/dedicated-environments/overview.md

```diff
@@ -0,0 +1,69 @@
+
+For a full  list of public regions and IP addresses, visit the [Regions page](https://docs.platform.sh/development/regions.html#regions).
+
+In a Grid region, incoming and outgoing traffic is handled via central region gateways, and [publicly available IP addresses](https://docs.platform.sh/development/regions.html#public-ip-addresses) can be used for external firewalls. The public IP addresses for these public regions are stable but not guaranteed never to change.
+
+## **Grid vs Dedicated** 
+
+Whether you choose a Grid or Dedicated environment depends on the needs you have. In the table below, you can see the different ways in which either environment might work for you: 
+
+| FEATURE | GRID | DEDICATED |
+| --- | --- | --- |
+| **SLA** | 99.9% | 99.99% |
+| **Infrastructure** | Containers with dedicated resources on top of a shared redundant infrastructure | Dedicated 3 node clusters |
+| **Functioning** | A single container is deployed per runtimes and per services | Three nodes are running all applications and services are replicated across all of them |
+| **Resources allocation** | Resources are spread through one container with fixed sizes after deployment | Resources are deployed across 3 nodes |
+| **Usable regions** | Only the publicly available regions can be used | Any region can be used to deploy |
+| **Autonomous upsize** | Yes | Managed through Platform.sh |
+| **Upsize and downsize methods** | Redeploy - possible downtime depending on the hooks | No downtime - each instance is upsize in a rolling fashion |
+| **Multi-app support** | Supported natively | Supported through docroots with additional billing |
+| **Production branch** | Master or main branch | Dedicated production branch |
+| **Custom domains name** | Supported on master and any other branches | Supported on the production environment |
+| **Sync and merge functionalities** | Yes on all branches | Only on Development environments |
+| **Environment clone** | Yes on all branches | Only on Development environments |
+| **MySQL Replication** | None: standalone service container | Yes: 3 services nodes cluster |
+| **Redis Replication** | None: standalone service container | Yes: 3 services nodes cluster |
+| **MongoDB** | Standalone service container | Yes |
+| **CDN** | Fastly for master or main and pre production | Fastly |
+| **PHP version upgrade** | Autonomous | Autonomous |
+| **NodeJS version upgrade**| Autonomous | Autonomous |
+| **Routes management** | Autonomous | Autonomous |
+| **Crons management** | Autonomous | Autonomous |
+| **Crons tasks interrupted by deploys** | No: a running cron task will block a deployment until it is complete | Yes: a deploy will terminate a running cron task |
+| **Mounts management** | Autonomous | Either autonomous or manual |
```

Comment:
for dedicated we have "Either autonomous or manual" but I think what we mean here is autonomous or requires a support ticket" since the DG2 page states 

> Storage: Storage allocation between mounts, DB and services is done through Platform.sh once a ticket is raised. Storage management is not self-service.

Reply:
What if we make it

> Managed by Platform.sh (Dedicated Gen 2 only)

?